src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -10,92 +9,234 @@ import { NewSearchBarInput_viewer$data } from "__generated__/NewSearchBarInput_v
 import { NewSearchBarInputSuggestQuery } from "__generated__/NewSearchBarInputSuggestQuery.graphql"
 import createLogger from "Utils/logger"
 import { NewSearchInputPillsFragmentContainer } from "Components/Search/NewSearch/NewSearchInputPills"
-import { SuggestionItem } from "Components/Search/Suggestions/SuggestionItem"
 import { NewSearchBarFooter } from "Components/Search/NewSearch/NewSearchBarFooter"
 import { getLabel } from "./utils/getLabel"
+import { getSearchTerm } from "./utils/getSearchTerm"
 import { isServer } from "Server/isServer"
+import { PillType, TOP_PILL } from "Components/Search/NewSearch/constants"
+import {
+  NewSuggestionItem,
+  SuggionItemOptionProps,
+} from "./SuggestionItem/NewSuggestionItem"
+import { debounce } from "lodash"
+import { useTracking } from "react-tracking"
+import * as DeprecatedSchema from "@artsy/cohesion/dist/DeprecatedSchema"
+import { getENV } from "Utils/getENV"
+// eslint-disable-next-line no-restricted-imports
+import request from "superagent"
```

Comment:
Just use `fetch` This is a restricted import for a reason — we want to remove it completely.